### DISCUSSION TITLE
Suggestion for quick send to a specific folder with drag & drop.

### DISCUSSION THREAD

**Discussion by V0LDY** [localsend/localsend#1335] [Discussion (English, International)]
Hi, program is incredible (up to now I used Telegram to move files between devices, which is kinda ridiculous), but I have a suggestion that imho would make it even better, especially on Android devices where moving files between folders is a bit more cumbersome and having a way to put them directly in the right place would be so nice.

My suggestion is to add a drag&drop quick targets so that I can just send a file straight into the desired folder, which could be customized by the user depending on his needs. I'd try implementing it myself if my programming was good enough, unfortunately it's not so I hope some dev can pick up the suggestion!

This is a mockup of how something like that could look like.

![localsend mockup](https://github.com/localsend/localsend/assets/30637340/e6e9e1e0-89d9-4495-b321-a79b92687ed5)

![localsend annotated](https://github.com/localsend/localsend/assets/30637340/66d1b8db-aff8-43c6-b96d-972e0bec1c14)

**Comment by Tienisto** [comment]
This will be difficult because the protocol does not allow the sender to specify the destination

**Reply by V0LDY** [reply]
Is there a simple explanation for why? I'd like to understand, since in the app settings the user can change the destination folder where the files will end up, so I assumed it's not that big of a deal to have multiple options to choose.

Is it because the folder setting is fixed at the receiving end once the connection between the two devices is established? 
If that's the case, woulnd't it be possible to download the file in the stock folder then moving it automatically after the download is complete from the default folder to the desired one?
